include/bluetooth/uuid.h

```diff
@@ -79,6 +79,58 @@ struct bt_uuid_128 {
 #define BT_UUID_32(__u) CONTAINER_OF(__u, struct bt_uuid_32, uuid)
 #define BT_UUID_128(__u) CONTAINER_OF(__u, struct bt_uuid_128, uuid)
 
+
+/**
+ * @brief Encode 128 bit UUID into an array values
+ *
+ * UUID in 128 bit form requires an array creation.
+ * To complicate the whole thing - it requires the array to start from LSB,
+ * so using the readable form, we have to write it down backwards.
+ *
+ * `6E400001-B5A3-F393-E0A9-E50E24DCCA9E` have to written like below in
+ * the code:
+ *
+ * @code
+ * 0x9E, 0xCA, 0xDC, 0x24, 0x0E, 0xE5, 0xA9, 0xE0, \
+ * 0x93, 0xF3, 0xA3, 0xB5, 0x01, 0x00, 0x40, 0x6E
+ * @endcode
+ *
+ * This macro simplifies it by the usage of more user-friendly form:
+ *
+ * @code
+ * BT_UUID_128_ENC(0x6E400001, 0xB5A3, 0xF393, 0xE0A9, 0xE50E24DCCA9E)
+ * @endcode
+ *
+ * Just replace the hyphen by the comma and add `0x` prefixes.
+ *
+ * @param w32 First part of the UUID (32 bits)
+ * @param w1  Second part of the UUID (16 bits)
+ * @param w2  Third part of the UUID (16 bits)
+ * @param w3  Fourth part of the UUID (16 bits)
+ * @param w48 Fifth part of the UUID (48 bits)
+ *
+ * @return The comma separated values for UUID 128 initializer that
+ *         may be used directly as an argument for
+ *         @ref BT_UUID_INIT_128 or @ref BT_UUID_DECLARE_128
+ */
+#define BT_UUID_128_ENC(w32, w1, w2, w3, w48) \
```

Comment:
The abbreviation `_ENC` as short might be confused with encryption (mbedtls, or for link layer). I would suggest avoiding `_ENC` here. Possibly use the full name `_ENCODE`, or maybe `BT_UUID_VAL_INIT_128`

Reply:
Maybe `BT_UUID_128_ARRAY`. I wish to avoid another `INIT`, as we have `BT_UUID_INIT_128` already. It would look just silly:
```
BT_UUID_INIT_128(BT_UUID_VAL_INIT_128(...))
```
Anyway - I found `_ENCODE` be also good - so it is only the matter of decision.

Reply:
Ok. then maybe just go with `_ENCODE`.

Reply:
Done